sentry-samples/kmp-app/iosApp/iosApp/ContentView.swift

```diff
@@ -4,6 +4,17 @@ import shared
 struct ContentView: View {
 	var body: some View {
 		Text("KPM Sample App " + Platform().platform)
+        VStack() {
+            Button("Capture Message") {
+                SentryKMP().captureMessage(message: "KMP Sample App " + Platform().platform)
+            }
+            Button("Capture Exception") {
+                SharedBusinessLogic().doException()
+            }
+            Button("Hard Crash") {
+                SharedBusinessLogic().hardCrash()
```

Comment:
Why not using `SentryKMP`?

Reply:
I wanted the sample to show how exceptions thrown in the shared layer look like (especially the stack trace), but I could also add some more examples using SentryKMP directly